fala/apps/adviser/tests/test_other_regions_view.py

```diff
@@ -24,6 +22,7 @@ def test_region_postcodes(self):
             {"postcode": "im4", "message": "The postcode IM4 is in the Isle of Man"},
             # Invalid post code, results are found and search
             {"postcode": "AB11 9EE", "message": "Enter a valid postcode"},
+            {"postcode": "IM4 TESTTTTTTTTTTTT", "message": "Enter a valid postcode"},
```

Comment:
Can you add a scotland postcode that actually is England into this test.

`TD15 1UY` and `TD15`

Reply:
Done